Robus/src/reception.c

```diff
@@ -53,21 +53,23 @@ void Recep_Init(void)
  ******************************************************************************/
 void Recep_GetHeader(volatile unsigned char *data)
 {
-    ctx.tx_lock = true;
```

Comment:
This is not related to CRC but to TX_lock

Reply:
still there just change position